type-definitions/dom-event.d.ts

```diff
@@ -0,0 +1,46 @@
+import {Stream} from "most";
+
+export function domEvent(event: string, node: EventTarget, capture: boolean): Stream<Event>
```

Comment:
Ah darn. I guess default values aren't allowed here. Ok, let's go with your original plan of just making it optional.